pyat/at/load_mat.py

```diff
@@ -3,23 +3,146 @@
 
 This is working from a specific file and may not be general.
 """
-
 import scipy.io
 from . import elements
 import numpy
 
 
-CLASS_MAPPING = {'Quad': 'Quadrupole',
-                 'Sext': 'Sextupole'}
+CLASS_MAPPING = {'Quad': 'Quadrupole', 'Sext': 'Sextupole', 'AP': 'Aperture',
+                 'RF': 'RFCavity', 'BPM': 'Monitor'}
+
+CLASSES = set(['Marker', 'Monitor', 'Aperture', 'Drift', 'ThinMultipole',
+               'Multipole', 'Dipole', 'Bend', 'Quadrupole', 'Sextupole',
+               'Octupole', 'RFCavity', 'RingParam', 'M66', 'Corrector'])
 
-FAMILY_MAPPING = {'AP': 'Aperture'}
+PASSMETHOD_MAPPING = {'CorrectorPass': 'Corrector', 'Matrix66Pass': 'M66',
+                      'CavityPass': 'RFCavity', 'QuadLinearPass': 'Quadrupole',
+                      'BendLinearPass': 'Bend', 'AperturePass': 'Aperture',
+                      'ThinMPolePass': 'ThinMultipole', 'DriftPass': 'Drift'}
 
 
-def load_element(element_array):
+def hasattrs(kwargs, *attributes):
```

Comment:
The implementation suggests this will return True if any of the attributes are present in kwargs, but the comment doesn't make that clear.